packages/components/plh/plh-kids-kw/components/module-list-item/module-list-item.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit } from "@angular/core";
+import { FlowTypes } from "packages/data-models";
+import { TemplateBaseComponent } from "src/app/shared/components/template/components/base";
+import { TemplateTranslateService } from "src/app/shared/components/template/services/template-translate.service";
+import {
+  getBooleanParamFromTemplateRow,
+  getStringParamFromTemplateRow,
+} from "src/app/shared/utils";
+
+interface IModuleListItemParams {
+  /* TEMPLATE PARAMETER: "module_image_asset". The image attached to the module */
+  moduleImageAsset: string | null;
+  /* TEMPLATE PARAMETER: "text_transform". The format of the text on the module item. Default null */
+  textTransform: "capitalise" | "uppercase" | null;
```

Comment:
I think we probably don't need the `capitalise` option. It's going to apply a sub-optimal version of title case where words like "of", "to", "and" etc. get capitalised, a convention which may also vary by language. So I think capitalisation within strings is something that authors need to determine themselves when supplying the string. The same argument could be applied, to some extent, to "uppercase" too, but I can see how this could be useful so I'm happy to have it included.